src/Equinox/Core.fs

```diff
@@ -31,6 +31,8 @@ and [<NoEquality; NoComparison; RequireQualifiedAccess>] SyncResult<'state> =
 /// Store-specific opaque token to be used for synchronization purposes
 and [<Struct; NoEquality; NoComparison>] StreamToken = { value: obj; version: int64; streamBytes: int64 }
 
+open Tracing
```

Comment:
why do people always have to open things ;) best to be explicit at consume site

Reply:
It's extension methods, no choice but to open :(

Reply:
Rider definitely feels the need to open things

Reply:
There's more to life than pressing Alt Enter all the time bro

Reply:
You don't even need to press alt+enter anymore, You just type Activity and it imports it for you 😂

Reply:
well type slower and turn off your fancy gizmos then :D